cogs/util.py

```diff
@@ -0,0 +1,127 @@
+from discord.ext import commands
+import discord
+
+
+class Util(commands.Cog):
+    """
+    Utility commands
+    """
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @commands.command(name="ping")
+    async def ping(self, ctx: commands.Context) -> None:
+        """
+        Get the bot's latency, in milliseconds
+        """
+        await ctx.send(f"Pong 🏓! Latency was {int(self.bot.latency * 1000)} ms")
+
+    @commands.command(name="reload")
+    async def reload(self, ctx: commands.Context, *cogs: str) -> None:
+        """
+        Reload cogs by name, or `all`
```

Comment:
it looks like the ` breaks the help command
![image](https://github.com/SJSUCSClub/acm-bot/assets/55469119/246af17a-2f86-4a02-94d2-0aaecfe73a6d)

Reply:
Yeah, apparently that is an issue. For now, I changed the docstrings to not use the ` (backtick) in their brief descriptions.